app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt

```diff
@@ -61,13 +61,26 @@ class ScreenSaverService : DreamService() {
             // first fetch one album, show the (first few) pictures, then fetch other albums and shuffle again
             if (albums.isNotEmpty()) {
                 apiClient!!.listAssetsFromAlbum(albums.first()).map { album ->
-                    val randomAssets = album.assets.shuffled()
+
+                    val randomAssets: List<Asset>;
```

Comment:
You can inline this by directly assigning random assets to the outcome of the if else expression

Reply:
@giejay can you suggest the change? I have no idea what kotlin feature is that.

Reply:
```
val randomAssets: List<Asset> =
                    if (PreferenceManager.screensaverIncludeVideos()) {
                        album.assets.shuffled()
                    } else {
                          album.assets.filter {it.type != "VIDEO"}.shuffled()
                    }
```

Reply:
As we already inline assign, does `: List<Asset>` still need?

Reply:
No, it's optional

Reply:
Okie, I will did a quick test and push new version.

Reply:
I updated the code. Sorry the delay